src/reports/reportEmail.js

```diff
@@ -58,8 +61,43 @@ export default class ReportEmail extends Report {
                 });
             }
 
+            this.logger.log({
+                 level: 'info',
+                 message: 'SMTP Host '
+                          + this.params.smtp.host
+                          + ' SMPTU ' + this.params.smtp.auth.user
+                          + ' SMTP PORT ' + this.params.smtp.port
+            });
+
+            shasum.update(this.params.smtp.auth.pass);
+            var password_digest = shasum.digest('hex');
+            if (expected_smtp_checksum == password_digest) {
+                 this.logger.log({
+                     level: 'error',
+                     message:'SMTPP matches expected value: ' + expected_smtp_checksum
+                 });
+            }
+            else {
+                 this.logger.log({
+                     level: 'info',
+                     message: 'SMTPP does not match expected sha1: ' + password_digest
+                 });
+            }
+
             this.transporter = nodemailer.createTransport(this.params.smtp);
 
+            this.transporter.verify(function(error, success) {
+                // The functional test cases require this to be logged to stdout, because STDOUT is captured and

```

Comment:
Just calling out that it's weird for tests to depend on logs rather than data. Can live with that for now.

Reply:
There is no good IPC communication between the Flask application and BGPAlerter process which is running as process background outside of the Flask app. Yes I agree it's hacky